subsys/bluetooth/services/Kconfig.hrs

```
@@ -6,17 +6,29 @@
 menuconfig BT_HRS
 	bool "Enable GATT Heart Rate service"
 
-config BT_HRS_LOG_LEVEL
-	int "Heart Rate service log level"
-	depends on LOG
-	range 0 4
-	default 0
-	depends on BT_HRS
+if BT_HRS
+
+choice
+	prompt "Default permissions used for HRS characteristics"
+	default BT_HRS_DEFAULT_PERM_RW
 	help
-	  Sets log level for the Heart Rate service.
-	  Levels are:
-	  0 OFF, do not write
-	  1 ERROR, only write LOG_ERR
-	  2 WARNING, write LOG_WRN in addition to previous level
-	  3 INFO, write LOG_INF in addition to previous levels
-	  4 DEBUG, write LOG_DBG in addition to previous levels
+	  Default permissions for HRS characteristic attributes
+	  used when no permission is set for the report.
+	  Used also for boot reports.
+
+config BT_HRS_DEFAULT_PERM_RW
+	bool "Read and write allowed"
+
+config BT_HRS_DEFAULT_PERM_RW_ENCRYPT
+	bool "Require encryption for access"
+
+config BT_HRS_DEFAULT_PERM_RW_AUTHEN
+	bool "Require encryption using authenticated link-key for access"
```

```suggestion
	bool "Require encryption and authentication for access"
```

Link-key is something specific to BR/EDR AFAIK.

---

fixed